integreat_cms/core/settings.py

```diff
@@ -64,6 +64,26 @@
 
 TEST_REGION_SLUG: Final[str] = "testumgebung"
 
+#: Reserved region slugs, that are used in the webapp
+RESERVED_REGION_SLUGS: Final[list[str]] = [
+    "landing",
+    "recommend",
+    "licenses",
+    "main-disclaimer",
+    "jpal",
+    "not-found",
+]
+
+#: Reserved region page patterns
+RESERVED_REGION_PAGE_PATTERNS: Final[list[str]] = [
+    "disclaimer",
```

Comment:
```suggestion
    IMPRINT_SLUG,
```
Since the imprint slug is configurable, we should also include it here.